### ISSUE TITLE
[Bug] Automatic centric function doesn't work correctly if the taskbar has too many icons 

### ISSUE THREAD

**Issue by Droyk** [ChrisAnd1998/TaskbarX#874]
This also makes the scrolling next to impossible coz whenever the user tries to use the scroll bar using the arrow key or scroll wheel to see the hidden icons the automatic centric behavior immediately makes the scroll bar jumps to its default position.

See it in action: https://gfycat.com/ifr/ableobedientcoqui?hd=1&speed=2

Edition - Windows 10 Pro
Version - 21H1
OS build - 19043.1415
Experience - Windows Feature Experience Pack 120.2212.3920.0

I tried other taskbar centering programs such as https://github.com/mdhiggins/CenterTaskbar same result!

**Comment by ChrisAnd1998** [comment]
@Droyk Marking at as Can't fix for now. As I have no idea how I can fix this :/

**Comment by Droyk** [comment]
@ChrisAnd1998 Does that mean this happening on your PC too! or you can't reproduce it?

**Comment by Droyk** [comment]
Also, if this problem can't be solved then it makes all the more reason to add this feature #156 so the users don't have to disable TaskbarX main function that **Automatically Centers** the icons.

**Comment by ChrisAnd1998** [comment]
@Droyk Yes, this will happen for everyone because the taskbar works that way. There should also be an option that disables the centering when the icons hit the end of the taskbar. TaskbarX only knows the size of the taskbar by looking for the first and last icon in the taskbar. and get their position. If the last icon breaks to a new row the formula is broken. I will look into your suggestion of making them smaller ;)
